build.gradle

```diff
@@ -24,6 +24,7 @@ plugins {
 }
 
 apply plugin: 'io.github.robwin.jgitflow'
+apply plugin: 'maven'
```

Comment:
I added this [plugin](https://docs.gradle.org/current/userguide/maven_plugin.html) so that local builds can share dependencies with other local builds.  For example, I want to build common locally and use those changes in ingest service.  Running the `gradle install` will now publish libraries to your local .m2.  Please let me know if there is a better way to share dependencies from local builds.

Reply:
I'd recommend using the gradle "maven-publish" plugin that is already being used to publish to artifactory. It also supports a task for publishing locally. https://docs.gradle.org/current/userguide/publishing_maven.html#publishing_maven:install

Reply:
Sounds good.  I figured this may be due to me being a bit of a noob to Gradle.  I will remove this plugin.